### DISCUSSION TITLE
Script/CSS bundling issue

### DISCUSSION THREAD

**Discussion by kilroyFR** [serenity-is/Serenity#5999] [Help]
hey there, i am using Serenity 5.0.26 and was trying to configure the Script/CSS bundling in appsettings.json.

```
  "ScriptBundling": {
    "Enabled": true,
    "Minimize": true,
    "UseMinJS": true,
    "NoMinimize": [
      "~/Serenity.Assets/Scripts/ckeditor/ckeditor.js"
    ],
```

but whatever i can change, the "wwwroot\Scripts\site\myapp.web.js"is not being minified.
I tried on old version of serenity and got same result. That should be very trivial operation but cannot make it working.
Anyone did already experienced this ?

Thanks for any help.

**Comment by edwardch** [comment]
Have you also got a file called `appsettings.Development.json` with settings for local dev by any chance? I wonder if that might be interfering.
